app/client/cli/utils.go

```diff
@@ -144,6 +146,16 @@ func readPassphrase(currPwd string) string {
 	return credentials(currPwd)
 }
 
+func readPassphraseMessage(currPwd, prompt string) string {
```

Comment:
@deblasis is adding a non-interactive flag in https://github.com/pokt-network/pocket/pull/522/files, so I think we might be able to get around without needing this function at all

Reply:
@Olshansk This function is only for updating the passphrase - I just wanted it to say `Enter New Passphrase` in the prompt - not done much CLI stuff so I was kind of guessing what the end user would really want

Reply:
Okay, I understand what you're aiming for here. Sgtm